tuna/example/example_lib.py

```diff
@@ -98,25 +104,25 @@ def launch_worker(self, gpu_idx, f_vals, worker_lst):
     worker_lst.append(worker)
     return True
 
-  def compose_worker_list(self, machines):
+  def compose_worker_list(self, machines) -> Optional[List[ExampleWorker]]:
     # pylint: disable=too-many-branches
     """! Helper function to compose worker_list
-      @param res DB query return item containg available machines
-      @param args The command line arguments
+      @param machines containg available machines
+      @returns list of actual available machine list
```

Comment:
Unclear what this is describing.
machines - list of available machine objects
return - list of worker objects